src/main/java/Bob/Ui.java

```diff
@@ -0,0 +1,53 @@
+package bob;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Scanner;
+
+import bob.task.Task;
+
+public class Ui {
+
+    /**
+     * Prints a dialogue.
+     *
+     * @param input
+     */
+    public static void dialogue(String input) {
```

Comment:
dialogue is really vague, what kind of dialogue? Perhaps dialogueFormat would be better?

Reply:
I agree with Julia that `dialogue` as a variable name is vague. In this context, maybe `printDialogue` might be better?